main.py

```diff
@@ -990,7 +102,7 @@ def _should_ignore(self, file):
                             # Duplicate the line to retain the comment syntax.
                             new_line = file_lines[line_number]
                             remove = fr'{raw_issue.identifier}.*{raw_issue.title}'
-                            insert = f'Issue URL: {client.line_base_url}{client.repo}/issues/{new_issue_number}'
+                            insert = client.get_issue_url(new_issue_number)
```

Comment:
As noted in my earlier comment, this is just to pull the GitHub-specific URL format out of this function. Each client is now responsible for implementing this method as appropriate for their system.